.travis.yml

```diff
@@ -1,13 +1,15 @@
 language: node_js
-node_js:
-  - "4"	
-  - "6"	
-  - "8"
-  - "10"
 git:
   depth: 5
 cache: yarn
 branches:
   only:
     - master
-    - v5
+matrix:
```

Comment:
Could you please tell me the significance of using matrix?

Reply:
Matrix allows us to do different things in different Travis jobs. My understanding is that defining multiple versions of Node is effectively shorthand for defining a Matrix. But in this PR I expanded that into the longhand so I could add another job that performs the same test but sends code coverage results.